docs/source/articles/system-emails.html.md

```diff
@@ -5,7 +5,7 @@ excerpt: System emails are sent from both the admin and storefront engines. All
 
 # System Emails
 
-System emails are sent from both the admin and storefront engines. All system emails may be customized like any other view in the system, by first [overriding](overriding.html) the mailer views and associated stylesheets.
+System emails are sent from both the admin and storefront engines. All system emails may be customized like any other view in the system, by first [overriding](overriding.html) the mailer views and associated stylesheets. Emails sent from the system use Rails' [Action Mailer](https://guides.rubyonrails.org/action_mailer_basics.html#) to do all the heavy lifting, so if you're making a new mailer, be sure to read that guide for more information.
```

Comment:
```suggestion
System emails are sent from both the admin and storefront engines. All system emails may be customized like any other view in the system, by first [overriding](overriding.html) the mailer views and associated stylesheets. Emails are sent from the system use Rails' [Action Mailer](https://guides.rubyonrails.org/action_mailer_basics.html#) to do all the heavy lifting, so if you're making a new mailer, be sure to read that guide for more information.
```

Reply:
minor, but proper noun possessive use `'s', so it should be Rails's

https://www.proofreadnow.com/blog/bid/84201/Possessives-Proper-Nouns